R/AbstractGraphReporter.R

```diff
@@ -420,9 +420,14 @@ AbstractGraphReporter <- R6::R6Class(
                 visNetwork::visHierarchicalLayout(sortMethod = "directed"
                                                   , direction = "UD") %>%
                 visNetwork::visEdges(arrows = 'to') %>%
-                visNetwork::visOptions(highlightNearest = list(enabled = TRUE
-                                                               , degree = nrow(plotDTnodes) # guarantee full path
-                                                               , algorithm = "hierarchical"))
+                visNetwork::visOptions(
+                    highlightNearest = list(
+                        enabled = TRUE
+                        , degree = nrow(plotDTnodes) # guarantee full path
+                        , algorithm = "hierarchical"
+                    )
+                    , nodesIdSelection = TRUE
```

Comment:
What?! That's all that was needed for the node selection feature?  It is a great addition.